### PR TITLE
VDB-1001 use vdb InsertionModel for vow fess

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

File: transformers/events/deal/converter.go [hunk 1]
```diff
@@ -57,8 +57,8 @@ func (c Converter) ToModels(_ string, logs []core.HeaderSyncLog, db *postgres.DB
 				constants.LogFK,
 			},
 			ColumnValues: event.ColumnValues{
-				constants.HeaderFK:      log.HeaderID,
-				Id:                      bidId.String(),
+				constants.HeaderFK: log.HeaderID,
+				Id:                 bidId.String(),
```
Comment (rmulhol) [inline transformers/events/deal/converter.go]: Possibly needs a `gofmt ./...`?

File: transformers/events/vow_fess/converter.go [hunk 2]
```diff
@@ -19,38 +19,34 @@ package vow_fess
...
 
-		model := shared.InsertionModel{
+		model := event.InsertionModel{
 			SchemaName: "maker",
-			TableName:  "vow_fess",
-			OrderedColumns: []string{
-				constants.HeaderFK, "tab", constants.LogFK,
+			TableName:  constants.VowFessLabel,
+			OrderedColumns: []event.ColumnName{
+				constants.HeaderFK, constants.TabColumn, constants.LogFK,
```
Comment (m0ar) [inline transformers/events/vow_fess/converter.go]: `event.HeaderFK` etc

File: transformers/test_data/vow_fess.go [hunk 3]
```diff
@@ -52,16 +53,15 @@ var VowFessHeaderSyncLog = core.HeaderSyncLog{
...
+var VowFessModel = event.InsertionModel{
 	SchemaName: "maker",
 	TableName:  "vow_fess",
-	OrderedColumns: []string{
+	OrderedColumns: []event.ColumnName{
 		constants.HeaderFK, "tab", constants.LogFK,
 	},
-	ColumnValues: shared.ColumnValues{
+	ColumnValues: event.ColumnValues{
 		"tab":              "1337",
```
Comment (m0ar) [inline transformers/test_data/vow_fess.go]: :sunglasses:

File: transformers/events/vow_fess/converter.go (hunk 2 above)
Comment (elizabethengelman) [reply transformers/events/vow_fess/converter.go]: 👍 nice catch!

**Pull request merged**